src/main/java/com/thealgorithms/dynamicprogramming/MinimumSumPartition.java

```diff
@@ -1,89 +1,50 @@
 package com.thealgorithms.dynamicprogramming;
 
-// Partition a set into two subsets such that the difference of subset sums is minimum
+import java.util.Arrays;
 
 /*
-Input:  arr[] = {1, 6, 11, 5}
-Output: 1
+Given an array of non-negative integers , partition the array in two subset that
```

Comment:
If it is only for non-negative numbers, please check the input array and throw `IllegalArgumentException` when necessary, cf. https://github.com/TheAlgorithms/Java/pull/4368#issuecomment-1719945280